Source/Core/Core/PatchEngine.cpp

```diff
@@ -196,9 +196,6 @@ static void ApplyPatches(const std::vector<Patch> &patches)
 				case PATCH_32BIT:
 					Memory::Write_U32(value, addr);
 					break;
-				default:
```

Comment:
The same thing this code would have done either way. I can add a PATCH_INVALID type to it, and sanitize it if you really want me to.